src/main/java/com/github/wovnio/wovnjava/SubdomainUrlLanguagePatternHandler.java

```diff
@@ -19,7 +19,16 @@ Lang getLang(String url) {
         return (lang != null && this.supportedLangs.contains(lang)) ? lang : null;
     }
 
-    String removeLang(String url, String lang) {
+    String convertToDefaultLanguage(String url) {
```

Comment:
Could be a candidate for moving the implementation to the base class since it seems like it will always be the same, but will leave this up to you.

Reply:
That could be possible, but it references `removeLang` which I've changed to a private method inside the implementations. I think I'll leave it as it is because I don't want to force all pattern handlers to have a `removeLang` method, and it way gives me better flexibility to improve the internals of path query and subdomain handlers.